cartographer/mapping/probability_values.cc

```diff
@@ -23,12 +23,14 @@ namespace mapping {
 
 namespace {
 
+constexpr std::size_t kNumberOfCells = 32768;
+
 // 0 is unknown, [1, 32767] maps to [lower_bound, upper_bound].
 float SlowValueToBoundedFloat(const uint16 value, const uint16 unknown_value,
                               const float unknown_result,
                               const float lower_bound,
                               const float upper_bound) {
-  CHECK_LE(value, 32767);
+  CHECK_LT(value, kNumberOfCells);
   if (value == unknown_value) return unknown_result;
   const float kScale = (upper_bound - lower_bound) / 32766.f;
```

Comment:
What about this number?

Reply:
The condition was "less equal" and it is now "less than".

Reply:
I meant 32766.f in this line: `const float kScale = (upper_bound - lower_bound) / 32766.f`

Reply:
Done.

Reply:
While removing this, I was thinking to change the variable to `kNumberOfValues`.
WDYT?